app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -429,21 +370,32 @@ function toField(name: string, valueType?: ValueType | null | undefined): ColDef
     `
   const template =
     icon ?
-      `<span style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'><span ref="eLabel" class="ag-header-cell-label" role="presentation" style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'> ${name} </span>  ${menu} ${sort} ${svgTemplate}</span>`
-    : `<span ref="eLabel" style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'>${name} ${menu} ${sort}</span>`
+      `<span style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'><span ref="eLabel" class="ag-header-cell-label" role="presentation" style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'> ${name} </span>  ${menu} ${sort} ${getSvgTemplate(icon)} ${svgTemplateWarning}</span>`
+    : `<span ref="eLabel" style='display:flex; flex-direction:row; justify-content:space-between; width:inherit;'>${name} ${menu} ${sort} ${svgTemplateWarning}</span>`
+
   return {
     field: name,
     headerComponentParams: {
       template,
       setAriaSort: () => {},
     },
+    tooltipComponent: TableVisualisationTooltip,
     headerTooltip: displayValue ? displayValue : '',
+    tooltipComponentParams: {
+      numberOfNothing: index != null ? dataQuality.number_of_nothing[index] : null,
+      numberOfWhitespace: index != null ? dataQuality.number_of_whitespace[index] : null,
+      total:
+        typeof props.data === 'object' && 'data_quality_pairs' in props.data ?
```

Comment:
Is this condition necessary? I don't see a reason not to pass the total unconditionally, which would avoid the need to ensure this condition remains consistent with the logic deciding whether to show the data quality information.